public/resources/remix/style.less

```diff
@@ -272,3 +272,21 @@
     }
   }
 }
+/* move the remix button to the left side for AFrame projects only*/
+.a-body .details-bar.touch-mode {
+  right: auto !important;
+  left: 6px !important;
+    
+  .thimble-button {
+    left: 0px !important;
+  }
+    
+  .details-bar-remix-button-wrapper {
+    margin-right: auto !important;
+    margin-left: 46px !important;
+  }
+
+  .close-details-bar {
+    left: 10px !important;
+  }
+}
```

Comment:
For the benefit of others, having a `\n` newline at the end of the file vs. just having it end is done so that you avoid issues when files are concatenated and joined together at build time: you don't want the end of one file and the start of another to get interpreted as a single line in some cases.  Always having a line break on your last line is how we fix this.